tests/coverage/attr/trait-impl-inherit.rs

```diff
@@ -0,0 +1,23 @@
+// Checks that `#[coverage(..)]` in a trait method is not inherited in an
+// implementation.
+//@ reference: attributes.coverage.trait-impl-inherit
+
```

Comment:
Nitpick: For new coverage tests I usually also add `//@ edition: 2021` even if it's not relevant to the specific test, so that I don't have to worry about older-edition quirks.

Reply:
Sure, done!